Dockerfile

```diff
@@ -1,32 +1,66 @@
 # DockerFile for grpc4bmi. Installs the C++ bindings in the /usr/local prefix directory in the container. If you are
 # planning to run a container with your BMI-enabled model and communicate with it using grpc4bmi, you can use this as a
 # base image for your model
-FROM ubuntu:bionic
-MAINTAINER Gijs van den Oord <g.vandenoord@esciencecenter.nl>
-RUN apt-get update
+FROM ubuntu:24.04
+LABEL maintainer="eWaterCycle <ewatercycle@esciencecenter.nl>"
+LABEL org.opencontainers.image.source="https://github.com/eWaterCycle/grpc4bmi"
+
+ENV GRPC_VERSION="1.66.1"
+ENV BMIC_VERSION="2.1.2"
+ENV BMICXX_VERSION="2.0.2"
 
 # Prerequisite packages
-RUN apt-get install -y wget git build-essential g++ make cmake curl automake libtool pkg-config gfortran
+RUN apt-get update && apt-get install -y \
+    automake \
+    build-essential \
+    cmake \
+    curl \
+    g++ \
+    gfortran \
+    git \
+    libtool \
+    libssl-dev \
+    make \
+    pkg-config \
+    vim-tiny \
+    wget \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # Build grpc from source
-RUN git clone -b $(curl -L https://grpc.io/release) --depth=1 https://github.com/grpc/grpc /opt/grpc
+RUN git clone -b v${GRPC_VERSION} --depth 1 https://github.com/grpc/grpc /opt/grpc
 WORKDIR /opt/grpc
 RUN git submodule update --init --recursive
```

Comment:
Replacing these three layers with one, which allows me to remove the `.git` directory:
```dockerfile
RUN git clone --branch v${GRPC_VERSION} --depth 1 https://github.com/grpc/grpc /opt/grpc && \
    cd /opt/grpc && \
    git submodule update --init --recursive && \
    rm -rf /opt/grpc/.git
```
drops the final image size from 3.1 GB to 1.4 GB.

Would this be OK? I don't think there's a downside.